src/cli/fo_nomos_license_list.php

```diff
@@ -27,11 +32,16 @@
   -u upload id        :: upload id
   -t uploadtree id    :: uploadtree id
   -c sysconfdir       :: Specify the directory for the system configuration
+  --type export type  :: For License: license (default), For Copyright: copyright
   --username username :: username
   --password password :: password
   --container         :: include container or not, 1: yes, 0: no (default)
-  -x                  :: do not show files which have unuseful license 'No_license_found' or no license
-  -X excluding        :: Exclude files containing [free text] in the path.
+  -x                  :: Licence from files which do not have unuseful license 'No_license_found' or no license
+  -y                  :: Copyrights from files which do not have license
+                         Files without license refers to:
+                         File had no license finding by agents and no license was added by users. 
+                         File had license findings by agents but were either removed or file was marked as irrelevant.
+  -X excluding        :: Exclude files containing [free text] in the path
```

Comment:
Possible irrelevant change.

Reply:
Hi Sir, I am not very clear with this. Please explain me.

Reply:
While providing a patch, it is best practice to not touch lines which you are not working on. For example here, you removed the `.` from the end of line making it also the part of your commit/PR even tough, you really did not wanted to do something with it.

Reply:
And please, you need not call us sir/ma'am 😃